kubevirtci

```diff
@@ -203,6 +212,12 @@ function kubevirtci::install_capk_release {
 
 function kubevirtci::install_calico {
 	kubevirtci::kubectl_tenant apply -f https://docs.projectcalico.org/${CALICO_VERSION}/manifests/calico.yaml
+    echo "Waiting for calico pods rollout"
+	while [[ "$(kubevirtci::kubectl_tenant get ds -n kube-system --no-headers | grep -v Found | grep calico-node | wc -l)" -eq 0 ]]; do
+		sleep 5
+	done
```

Comment:
why do you need this? The calico-node DS is a part of the calico.yaml manifest which is installed a few lines above. The DS should immediately exist and not need to be waited on just for existence.

Reply:
Right, removing it.